bundle-workflow/src/run_integ_test.py

```diff
@@ -29,13 +29,16 @@
 def parse_arguments():
     parser = argparse.ArgumentParser(description="Test an OpenSearch Bundle")
     parser.add_argument(
-        "--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file."
+        "--s3-bucket", type=str, help="S3 bucket name"
```

Comment:
As we are taking S3 bucket name, I think we should also let users configure the role to pull these artifacts. And the default could be `opensearch-test-role`.

Reply:
that's configurable via environment variables.